components/inputs/inputs.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+@import url("../../styles/doc.css");
+
+
+.btn-wrapper{
+    margin-top: 3rem;
+}
+
+.input-form{
+    /* background-color: aqua; */
+}
+
+.required-field label::after{
+    content: "*";
+    color: red;
+}
+.email-input, textarea{
+    border-radius: 5px;
+    width: 45rem;
+    border: 2px solid #dfdfdf;
+    margin-bottom: 15px;
+    padding: 10px;
+}
+
+.required{
+    color: var(--primary-color);
+    font-weight: bold;
+}
+
+.btn{
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    padding: 12px;
+    border-radius: 5px;
+    /* margin:1rem; */
+}
```

Comment:
In btn padding try to use rem if it is more than 10px ... as vikram sir also suugested this in his css session..  An also try to delete commented code ...